Java Script/index.js

```diff
@@ -0,0 +1,81 @@
+/*Loop for adding all the numbers bettwen 0 and 1000 with break > 400*/
+let num = 0;
+let sum = 0;
+for (num = 0; num < 1000; num++) {
+    sum += +num;
+    if (sum > 400) break;
+}
+alert("Sum: " + sum);
+
+/*Loops: Show numbers from 0 to 10*/
+for (let currentNumber = 0; currentNumber < 10; currentNumber++) {
+    alert (`Number ${currentNumber}`);
+}
+console.log ("currentNumber: ", currentNumber)
+
+/*Loops: Show numbers from 100 to 0*/
+for (let currentNumber = 100; currentNumber > 0; currentNumber--) {
+    alert (`Number ${currentNumber}`);
```

Comment:
When your loop has many iterations, like 100, displaying information using alert is a bit hardcore 👩‍🎤 🎸 
each time(out of 100), alert pops up, you have to click it, then again and again. What if there's mistake at number 50
or at the end of a loop? It will be too frustrating to test your loop going through 100 alerts.
Better to use `console.log()` in those cases.

Reply:
This pokes me. Yesterday, my browser got frozen thanks to this error :|